Source/WebKitLegacy/mac/WebView/WebPreferences.h

```diff
@@ -209,12 +209,10 @@ WEBKIT_CLASS_DEPRECATED_MAC(10_3, 10_14)
 */
 @property (nonatomic) BOOL autosaves;
 
-#if !TARGET_OS_IPHONE
```

Comment:
This file is public API, I doubt you mean to be exposing new WebKitLegacy public API on iOS?

Reply:
I guess it's *not* public API on iOS...

Reply:
But it is on mac. But this is preprocessed out there? OK, nevermind!